cli/parameters/parameters.go

```diff
@@ -1,22 +1,12 @@
 package parameters
 
-import "github.com/spf13/cobra"
+import "fmt"
 
-type ParamError struct {
+type paramError struct {
 	Parameter string
 	Message   string
 }
 
-type Params interface {
-	Validate(cmd *cobra.Command, args []string) []ParamError
-}
-
-func ValidateParams(cmd *cobra.Command, args []string, params ...Params) []ParamError {
-	errors := make([]ParamError, 0)
-
-	for _, param := range params {
-		errors = append(errors, param.Validate(cmd, args)...)
-	}
-
-	return errors
+func (pe paramError) Error() string {
```

Comment:
in go, `error` is an interface, so implemeting this method makes the original `paramError` be treated as any other `error`. just this changes makes it compliant with the new validator approach [from here](https://github.com/kubeshop/tracetest/pull/2829/files#diff-1896d368b259c48ad87920f423ae9233acc2273dc63a314542ab7049fcb6b61cR60)